src/utils/getNetworkPrefix.ts

```diff
@@ -0,0 +1,8 @@
+function getNetworkPrefix(address = ''): string {
+  const splittedAddress = address.split(':')
```

Comment:
```suggestion
  const splitAddress = address.split(':')
```

("splitted" isn't a word)

Reply:
done!